examples/teleport-usage/teleport_usage.py

```diff
@@ -0,0 +1,115 @@
+import json
+import datetime
+import boto3
+import time
+import os
+
+from boto3.dynamodb.conditions import Key, Attr
+
+
+# Check for AWS DynamoDB table name and region ENV, if none prompt
+TABLE_NAME = os.getenv("TABLE_NAME")
+if TABLE_NAME == None:
+    TABLE_NAME = input("DynamoDB events table name: ")
+
+aws_region = os.getenv("AWS_REGION")
+if aws_region == None:
+    aws_region = input("AWS Region for DynamoDB: ")
+
+# Check ENV for start and end times, if none, set last 30 days
+start_date = os.getenv("START_DATE")
+if start_date == None:
+    now = datetime.datetime.now()
+    start_date = now - datetime.timedelta(days=30)
+    
+end_date = os.getenv("END_DATE")
+if end_date == None:
+    end_date = now

```

Comment:
`now` may an unbound var here. It only gets set above conditionally when START_DATE is not provided via env var.

Reply:
Fixed by 
```
end_date = os.getenv("END_DATE")
if not end_date:
    end_date = datetime.datetime.now()
else:
    end_date = datetime.datetime.strptime(end_date,'%Y-%m-%d')
```